packages/story-editor/src/components/library/panes/media/common/innerElement.js

```diff
@@ -117,7 +117,7 @@ function InnerElement({
 
   useEffect(() => {
     // assign display poster for videos
-    if (resource.poster && resource.poster.includes('blob')) {
+    if (resource.poster) {
```

Comment:
This fixes an issue with trimmed videos not showing poster images, even through one is set. Not sure why is this is happening, this it might be related to less state changes means less renders. With way, this works.

Reply:
At which moment is a trimmed video not showing poster image on `main`? Tried out the following:
- Add a video and trim it
- Wait until trimming finishes
- Check the Accessibility panel -- the poster image is visible.

Or did you mean something else?

Reply:
This is not a issue in main, just the PR. This is the only thing I could find to fix it. Without this change, poster is rendered by hidden for some reason.

Reply:
https://user-images.githubusercontent.com/237508/144619452-4190f548-2f2f-474b-a8bb-4f5dbae9b881.mov

This video, show the behaviour I am trying to fix. The poster is there but not shown.